api/types/custom-scalars/RawDraftContentState.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+const debug = require('debug')('api:types:custom-scalars');
+import { GraphQLScalarType, Kind } from 'graphql';
+import UserError from '../../utils/UserError';
+
+// Pretty dumb manual check to make sure a value is actual raw draft content state
+const isRawDraftContentState = (contentState: string) => {
+  let parsed;
+  try {
+    parsed = JSON.parse(contentState);
+  } catch (err) {
+    debug('error parsing raw draft content state');
+    return false;
+  }
+  if (!parsed.blocks || !Array.isArray(parsed.blocks) || !parsed.entityMap) {
+    debug('invalid raw draft content state');
+    return false;
+  }
+  return true;
+};
+
+const RawDraftContentState = new GraphQLScalarType({
```

Comment:
it's so cool! i quite no idea how to handle serialize JSON in GraphQL schema, until now😋